### PR TITLE
[spinel] Include pre-defines before verify defines

### PR DESCRIPTION
**Pull request by kghost**
Signed-off-by: Zang MingJie <mingjiez@google.com>

### CODE REVIEW

File: src/lib/spinel/openthread-spinel-config.h [hunk 1]
```diff
@@ -34,6 +34,8 @@
 #ifndef OPENTHREAD_SPINEL_CONFIG_H_
 #define OPENTHREAD_SPINEL_CONFIG_H_
 
+#include <openthread-core-config.h>
```
Comment (bukepo) [inline src/lib/spinel/openthread-spinel-config.h]: We intended to remove dependency on core config.

**Comment by kghost** [comment]
This is causing problems, since the define being checked in this file should be defined before this file.

How to resolve the problem ?

**Comment by bukepo** [comment]
I think you can define in CPPFLAGS.

**Comment by jwhui** [comment]
@kghost, any plan to move this PR forward?

**Comment by jwhui** [comment]
Closing stale PR.

**Pull request closed without merging**